### PR TITLE
Support the upcoming VAD params

### PR DESCRIPTION
**Pull request by mayeaux**
I believe this supports the upcoming API 

https://github.com/guillaumekln/faster-whisper/pull/95/files

Not 100% sure on my implementation but I think it's working OK for me locally

### CODE REVIEW

**Comment by jordimas** [comment]
Looks good to me. I will merge this as soon the new release of faster-whisper with this functionality is released

**Comment by jordimas** [comment]
Thanks for this!

**Comment by mayeaux** [comment]
Glad I could help! My Python skills aren't great so it was a good exercise for me lol, thanks for the module!

File: src/whisper_ctranslate2/whisper_ctranslate2.py [hunk 1]
```diff
@@ -185,6 +185,20 @@ def read_command_line():
...
+        "--vad_filter",
+        type=bool,
+        default=False,
+        help="Enable the voice activity detection (VAD) to filter out parts of the audio without speech. This step is using the Silero VAD model https://github.com/snakers4/silero-vad.",
+    )
+
+    parser.add_argument(
+        "--vad_min_silence_duration_ms",
+        type=float,
+        default=300,
```
Comment (escape0707) [inline src/whisper_ctranslate2/whisper_ctranslate2.py:198]: Wow, this default value is significantly shorter than the upstream default value (2s), which is unexpected, and it seems to cause the timing of subtitles to be unstable due to my experiences. Can we make this default value stays the same as the upstream one?

**Pull request merged**